nudge/src/email.ts

```diff
@@ -0,0 +1,30 @@
+import dotenv from 'dotenv'
+import request from 'request-promise-native'
+
+dotenv.config()
```

Comment:
Do you want to use the same config loading stuff used in other places?

Reply:
nah, I think this is fine for now. It will run in a dockerized environment with the configuration explicitly passed to it